quadproduction/OpenPype
# Add status on instance creator

## Changelog Description
Create a new "ftrack status" attribute for the creators that will set the status of the AssetVersion on ftrack. This attribute is an enum with only available status for the current project schema.

## Testing notes:
1. publish an asset
2. go in the publish instance settings 
3. set the status you want
4. publish
5. go on ftrack to see the status of the asset version changed
